.changes/unreleased/Feature-20221108-143458.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Feature
+body: Add support for adding expected tag key and tag value predicate to service ownership
```

Comment:
Can you split these into 2 different change entries.
```
Add support for adding expected tag key and tag value predicate to service ownership checks
```
and then
```
Add support for creating tags on teams
```

Easiest way it to modify this yaml file and then run `changie new` to add the second entry